source/js/components/join/join.jsx

```diff
@@ -133,6 +133,20 @@ export default class JoinUs extends React.Component {
     });
   }
 
+  renderGAEventLabel() {
```

Comment:
Since we are not actually "rendering" the label, let's change the method name to something else + make the name more specific as there are different GA events floating around in the file. Maybe something like `getGaSubmittedLabel`, `generateGaSubmittedLabel`, or `createGaSubmittedLabel`?